gateway/README.md

```diff
@@ -151,9 +151,9 @@ See the <a target="_blank" href="https://github.com/signalfx/gateway/blob/master
 |--------|----------|--------|
 | `Name` | A name for this forwarder. | "ourcarbon" |
 | `type` | The type of data that the gateway will emit. Possible values are `csv` for writing to a CSV file, `carbon` for sending to a Graphite server, and `signalfx` for sending to SignalFx. | "carbon" |
-| `BufferSize` | A performance configuration that specifies how many datapoints/events should be buffered in forwarders.  This is very useful when the gateway is configured with a `StatsDelay`. | `1000000` |
-| `MaxDrainSize` | A performance configuration that specifies the maximum amount of datapoints/events to emit in a single outbound request. This is very useful when the gateway is configured with a `StatsDelay`. | `5000` |
-| `DrainingThreads` | A performance configuration that specifies how many go routines should drain a buffer. This is very useful when the gateway is configured with a `StatsDelay`. | `50` |
+| `BufferSize` | A performance configuration that specifies the max datapoints/events to be buffered in forwarders.  | `1000000` |
```

Comment:
Is:  the max datapoints/events to be buffered
change to:  the maximum number of data points or events buffered